src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -13,66 +13,161 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using Microsoft.Quantum.Simulation.Core;
-using static Microsoft.Quantum.EntryPointDriver.Driver;
 
 namespace Microsoft.Quantum.EntryPointDriver
 {
+    using Validators = List<ValidateSymbol<CommandResult>>;
```

Comment:
I second going with immutable data structures by default, unless there is a good reason not to. Languages evolve, and their usage evolves. The default choice depends on the design of the code base, and this project is set up to use immutable data structures.

Reply:
changed to using ImmutableList